docs/spec/typeahead/typeahead.examples.async-data.use-case.md

```diff
@@ -0,0 +1,29 @@
+16.4: Typeahead Async data example
+==================================
+**Primary Actor**: User
+
+**Scope**: Ngx-bootstrap DEMO / BS version 3&4
+
+**Goal**: Show user Async data functionality
+
+Main success scenario:
+-------------------------
+1. User open Typeahead demo page
+2. User click on Async data sub-menu
+3. User is able to fill the "Model:" input. The input has a placeholder "Locations loaded with timeout"
+4. When user starts to type a name of a State from "statesComplex" array then a drop-down with a list of States matches is shown.
+Items quantity in the drop-down is limited with the key `typeaheadOptionsLimit`
```

Comment:
Items quantity in the drop-down is limited with the key `typeaheadOptionsLimit` - how?